aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -455,12 +444,7 @@ mod test {
     #[test]
     fn test_unknown_aggregator_disabled_fail() {
         let mut aggregator_data = AggregatorData::default();
-        assert_err!(aggregator_data.get_aggregator(
-            aggregator_id_for_test(200),
-            200,
-            &*TEST_RESOLVER,
-            false,
-        ));
+        assert_err!(aggregator_data.get_aggregator(aggregator_id_for_test(200), 200,));
```

Comment:
nit: no trailing comma, but actually test should be removed :)

Reply:
Done.